README.md

```diff
@@ -35,9 +35,10 @@ The module can be loaded to Linux kernel by runnning the command:
 $ sudo insmod vcam.ko
 ```
 
-Expectedly, two device nodes will be created in `/dev`:
+Expectedly, three device nodes will be created in `/dev`:
 * videoX - V4L2 device;
 * vcamctl - Control device for virtual camera(s), used by control utility `vcam-util`;
+* fbX - controlling framebuffer device;
```

Comment:
Then, we need to extend `vcam-util` to retrieve the device node of controlling framebuffer.

Reply:
At present, `vcam-util -l` dumps all information, but it would be great if a command option such as `--fbdev` shows the available name of controlling framebuffer. Then, we can program with the device.

Reply:
It's that means  `vcam-util -fbdev` showing framebuffer informations ?

Reply:
Yes, I meant the new option which outputs the framebuffer device name can help further applications for performing configurations. However, you can implement such feature in next pull request.